app/src/main/java/com/sunlightlabs/android/congress/fragments/BillInfoFragment.java

```diff
@@ -83,26 +80,6 @@ public void onActivityCreated(Bundle savedInstanceState) {
 	}
 	
 	public void setupControls() {
-		// if this was coming in from a search result and has associated highlight data, show it
-		if (bill.search != null && bill.search.highlight != null) {
-			
-			String field = Bill.matchField(bill.search.highlight);
-			
-			// don't bother showing the short title, or the official title if it's the official title being shown
-			if (field != null && !field.equals("popular_title") && !field.equals("keywords") && !field.equals("short_title") && !(field.equals("official_title") && bill.short_title == null)) {
-				View searchView = getView().findViewById(R.id.bill_search_data);
-				
-				String matchText = "\"" + bill.search.query + "\" matched the bill's " + Bill.matchText(field) + ":";
-				String highlightText = Utils.truncate(bill.search.highlight.get(field).get(0), 300, false);
-				if (field.equals("versions") || field.equals("summary"))
-					highlightText = "..." + highlightText + "...";
-				
-				((TextView) searchView.findViewById(R.id.match_field)).setText(matchText);
-				((TextView) searchView.findViewById(R.id.highlight_field)).setText(Html.fromHtml(highlightText));
-				
-				searchView.setVisibility(View.VISIBLE);
-			}
-		}
```

Comment:
The PP API doesn't support returning highlighted search excerpts, so this is removed. My feature request for it is here: https://github.com/propublica/congress-api-docs/issues/54